pantheon/src/main/java/tech/pegasys/pantheon/controller/PantheonControllerBuilder.java

```diff
@@ -151,6 +155,12 @@
     return this;
   }
 
+  public PantheonControllerBuilder<C> pendingTransactionRetentionPeriod(
```

Comment:
Again, adjacency.

Reply:
my bad but -- what do you mean by adjacency? maybe you mean move it near to `maxPendingTransactions`, but already it is

Reply:
prior to the force push it wasn't

Reply:
ah, yeah- I always have to force push after I rebase - is there a better way to do that?

Reply:
I tend to use `git pull upstream master --no-rebase` when pulling changes from master after I've created the PR the first time.  Avoids the need to force push.